pandas/core/arrays/timedeltas.py

```diff
@@ -397,6 +405,163 @@ def f(x):
 # ---------------------------------------------------------------------
 # Constructor Helpers
 
+def sequence_to_td64ns(data, copy=False, unit="ns", errors="raise"):
+    """
+    Parameters
+    ----------
+    array : list-like
+    copy : bool, default False
+    unit : str, default "ns"
+    errors : {"raise", "coerce", "ignore"}, default "raise"
+
+    Returns
+    -------
+    ndarray[timedelta64[ns]]
```

Comment:
is this how we indicate a multiple returns here? @datapythonista

Reply:
The right format IMO should be something like:
```
Returns
-------
data : numpy.ndarray
    The sequence converted to a numpy array of type ``timedelta64[ns]``.
inferred_freq : Tick or None
    The inferred frequency of the sequence.
```

We're adding this validation in #23432